scrapy/extensions/telnet.py

```diff
@@ -110,7 +105,7 @@ def _get_telnet_vars(self):
             'est': lambda: print_engine_status(self.crawler.engine),
             'p': pprint.pprint,
             'prefs': print_live_refs,
-            'hpy': hpy,
+            'hpy': None,
```

Comment:
I wonder what to do with this:
- Deprecate it somehow?
- Remove it?
- Use muppy?

In options 1 and 2, we might want to use muppy with a different name.

All of this is assuming muppy offers hpy or something similar, I’m not familiar with muppy or guppy at all.

Reply:
Can't we use muppy along with summary modules in pympler in place of guppy.hpy() as described in the documentation ?

Reply:
I think we should then just remove this key, instead of changing it to `None`.